src/vs/editor/browser/viewParts/lines/viewLines.css

```diff
@@ -26,6 +26,15 @@
 	-webkit-user-select: none;
 	-ms-user-select: none;
 }
+/* Use user-select: text for lookup feature on macOS */
+/* https://github.com/microsoft/vscode/issues/85632 */
+.monaco-editor.mac .lines-content:hover,
+.monaco-editor.mac .view-line:hover,
+.monaco-editor.mac .view-lines:hover {
+	user-select: text;
+	-webkit-user-select: text;
+	-ms-user-select: text;
+}
```

Comment:
I can confirm this working, by modifying local CSS through builtin dev tools.

But not without a catch: 

It seems that the whitespace indicators between words (`<span class="mtkw">·‌&zwnj;</span>`) may interfere with OS's tokenization, causing lookup to pick up the wrong word.

https://user-images.githubusercontent.com/1430856/182044406-58491d48-2bba-49e7-bc38-39b142f94e06.mov

Reply:
Good catch. I am afraid that we can't do much about OS tokenization.

You can try this HTML and see OS can tokenize `&npsp` but fail to tokenize the HTML with `&zwnj;` in one line.

![2022-08-01 15 57 54](https://user-images.githubusercontent.com/5123601/182101826-bd75f336-964c-4cbd-92a2-d18c26c392d6.gif)

[test-macos-lookup-tokenization.html.zip](https://github.com/microsoft/vscode/files/9232046/test-macos-lookup-tokenization.html.zip)

Reply:
Can you share the change here, just curious what it would be?

Reply:
MacOS won't treat `&zwnj;`(0x200C) as word separators. So we could add normal whitespace characters similar to https://github.com/microsoft/vscode/pull/150349